File: Java_ChildrenCompetition/src/main/java/model/TestParticipantRelation.java
package model;

public class TestParticipantRelation implements HasId<Tuple<Integer, Integer>> {
    Tuple<Integer, Integer> testParticipantID;

    public TestParticipantRelation(Tuple<Integer, Integer> testParticipantID) {
        this.testParticipantID = testParticipantID;
    }

    @Override
    public Tuple<Integer, Integer> getId() {
        return testParticipantID;
    }

    @Override
    public void setId(Tuple<Integer, Integer> id) {
        this.testParticipantID = id;
    }
}

File: CSharp_ChildrenCompetitionSockets/CSharp_ChildrenCompetitionSockets/model/TestParticipantRelation.cs
using System;

namespace CSharp_ChildrenCompetitionGUI.model
{
    public class TestParticipantRelation : HasId<Tuple<int, int>>
    {
        public Tuple<int, int> id { get; set; }

        public TestParticipantRelation(Tuple<int, int> id)
        {
            this.id = id;
        }
    }
}